### ISSUE TITLE
Problem with count matrix for Quality Check of Samples and Biological Replicates

### ISSUE THREAD

**Issue by perraudy** [trinityrnaseq/trinityrnaseq#1316]
Hello, I am trying to make the plots for quality control of my count matrix but every time I try to make any plot I get the same error message. I am following this pipeline: 
https://github.com/trinityrnaseq/trinityrnaseq/wiki/QC-Samples-and-Biological-Replicates

I am using this command line:
/scratch/project_2006096/Perraud/trinityrnaseq/Analysis/DifferentialExpression/PtR --matrix /scratch/project_2006096/Perraud/salmon/salmon.gene.counts.matrix --samples /scratch/project_2006096/Perraud/salmon/sample.txt --log2 --CPM --compare_replicates

Here are my error output after I run the job, my batch job file, the header of my count matrix and the sample list that is also asked in the pipeline.
[errors_17660788.txt](https://github.com/trinityrnaseq/trinityrnaseq/files/11966497/errors_17660788.txt)
[batchjob_plot_eachsamples.txt](https://github.com/trinityrnaseq/trinityrnaseq/files/11966501/batchjob_plot_eachsamples.txt)
[header_count_matrix.txt](https://github.com/trinityrnaseq/trinityrnaseq/files/11966502/header_count_matrix.txt)
[sample.txt](https://github.com/trinityrnaseq/trinityrnaseq/files/11966528/sample.txt)

Could someone help me figure out what is stopping the job? 
Thank you

**Comment by brianjohnhaas** [comment]
Hi,

It looks like the replicate VGbud_rep5 is missing from your counts matrix.

On Thu, Jul 6, 2023 at 5:19 AM perraudy ***@***.***> wrote:
>
> Hello, I am trying to make the plots for quality control of my count matrix but every time I try to make any plot I get the same error message. I am following this pipeline:
> https://github.com/trinityrnaseq/trinityrnaseq/wiki/QC-Samples-and-Biological-Replicates
>
> I am using this command line:
> /scratch/project_2006096/Perraud/trinityrnaseq/Analysis/DifferentialExpression/PtR --matrix /scratch/project_2006096/Perraud/salmon/salmon.gene.counts.matrix --samples /scratch/project_2006096/Perraud/salmon/sample.txt --log2 --CPM --compare_replicates
>
> Here are my error output after I run the job, my batch job file, the header of my count matrix and the sample list that is also asked in the pipeline.
> errors_17660788.txt
> batchjob_plot_eachsamples.txt
> header_count_matrix.txt
> sample.txt
>
> Could someone help me figure out what is stopping the job?
> Thank you
>
> —
> Reply to this email directly, view it on GitHub, or unsubscribe.
> You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>



-- 
--
Brian J. Haas
The Broad Institute
http://broadinstitute.org/~bhaas

**Comment by perraudy** [comment]
Indeed, thank you for pointing it out. I did all the analysis again with the VGbud_rep5 but I still have an error when I try to build the plots. It seems to be an error in the code it-self.
[errors_17701154.txt](https://github.com/trinityrnaseq/trinityrnaseq/files/11971451/errors_17701154.txt)
